### PR TITLE
Remove U2F support

### PR DESCRIPTION
**Pull request by Gusted**
- Completely remove U2F support from 1.18.0, 1.17.0 will be the last release that U2F is somewhat supported. Users who used U2F would already be warned about using U2F for a while now and should hopefully already be migrated to webauthn. But starting 1.18 definitely remove it.

### CODE REVIEW

**Comment by Gusted** [comment]
Note, there are other `// FIXME: DEPRECATED` for other related settings. But separating them so it become apparent in the changelog for 1.18.0 that support is removed.

File: routers/web/auth/webauthn.go [hunk 1]
```diff
@@ -67,10 +67,7 @@ func WebAuthnLoginAssertion(ctx *context.Context) {
 		return
 	}
 
-	// FIXME: DEPRECATED appid is deprecated and is planned to be removed in v1.18.0
-	assertion, sessionData, err := wa.WebAuthn.BeginLogin((*wa.User)(user), webauthn.WithAssertionExtensions(protocol.AuthenticationExtensions{
-		"appid": setting.U2F.AppID,
-	}))
+	assertion, sessionData, err := wa.WebAuthn.BeginLogin((*wa.User)(user))
```
Comment (lunny) [inline routers/web/auth/webauthn.go:70]: Could we remove that `appid`?

**Review by 6543 — APPROVED** [review]

File: routers/web/auth/webauthn.go (hunk 1 above)
Comment (Gusted) [reply routers/web/auth/webauthn.go:70]: Which appid? It's removed in the settings module.

Comment (Gusted) [reply routers/web/auth/webauthn.go:70]: As far as I know it's only required for U2F applications and Firefox+Chromium have deprecated this extension during login IIRC.

**Review by wxiaoguang — APPROVED** [review]
LGTM. 

ps: IIRC the locale files (except en-US) could be overwritten automatically to remove the deleted keys. Also it's fine to remove them manually.

**Pull request merged**